src/types.rs

```diff
@@ -248,6 +248,55 @@ impl From<JsonValue> for Value {
     }
 }
 
+impl From<Value> for JsonValue {
```

Comment:
Perhaps it makes more sense to implement `TryFrom` to avoid the usage of `unwrap` as much as possible? See my other comment

Reply:
sg. will do.